app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.habitsmasher"
-        minSdk 21
+        minSdk 26
```

Comment:
This change allows text to become auto scalable, and currently does not affect previous functionality.